<launch>
  <node pkg="rosbag" type="play" name="player"  args="--clock -l /home/jesus/rosbags/hand-90degMulti.bag"/>
 <node pkg="color_segmentation" type="color_segmentation_node" name="color_segmentation_node" output="screen">
   <remap from="/image" to="/hsrb/hand_camera/image_raw" />
   <!-- <remap from="/image" to="/hsrb/hand_camera/image_raw" /> -->
   <param name="term_distance" value="1000"/>
   <param name="epsilon" value="1.1"/>
   <param name="codebookSize" value="16"/>
   <param name="maxIter" value="10"/>
   <param name="debug" value="true"/>
   <param name="max_distance" value="25"/>
   <param name="min_distance" value="0"/>
   </node>
</launch>
